doc/electrs-rest-btcd-1.0.0.yaml

```diff
@@ -0,0 +1,936 @@
+      parameters: 
+        - name: address
+          in: path
+          required: true
+          schema: 
+            type: string
+      responses:
+        '200':
+          description: Transactions
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Transaction'
+  /address/{address}/txs/chain/{last_seen_txid}:
+    get:
+      tags:
+        - address
+      summary: Returns up to 50 mempool transactions plus the first 25 confirmed transactions.
+      description: Returns 25 transactions per page. More can be requested by specifying the last txid seen by the previous query.
+      parameters: 
+        - name: address
+          in: path
+          required: true
+          schema: 
+            type: string
+        - name: last_seen_txid
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Transactions
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Transaction'
+  /scripthash/{hash}/txs/chain:
+    get:
+      tags:
+        - scripthash
+      summary: Returns up to 50 mempool transactions plus the first 25 confirmed transactions.
+      description: Returns 25 transactions per page. More can be requested by specifying the last txid seen by the previous query.
+      parameters: 
+        - name: hash
+          in: path
+          required: true
+          schema: 
+            type: string
+      responses:
+        '200':
+          description: Transactions
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Transaction'
+  /scripthash/{hash}/txs/chain/{last_seen_txid}:
+    get:
+      tags:
+        - scripthash
+      summary: Returns up to 50 mempool transactions plus the first 25 confirmed transactions.
+      description: Returns 25 transactions per page. More can be requested by specifying the last txid seen by the previous query.
+      parameters: 
+        - name: hash
+          in: path
+          required: true
+          schema: 
+            type: string
+        - name: last_seen_txid
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Transactions
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Transaction'
+  /address/{address}/txs/mempool:
+    get:
+      tags:
+        - address
+        - mempool
+      summary: Get unconfirmed transaction history for the specified address/scripthash.
```

Comment:
omit information about scripthash (should be above)

Reply:
fixed both to be either address or script hash never both.